java/parser/src/main/java/org/sqlflow/parser/ParserGrpcServer.java

```diff
@@ -75,10 +79,10 @@ public void parse(ParserRequest request, StreamObserver<ParserResponse> response
 
       ParseResult parse_result;
       if (request.getDialect().equals("calcite")) {
-        CalciteParserAdaptor parser = new CalciteParserAdaptor();
+        ParseInterface parser = new CalciteParserAdaptor();
         parse_result = parser.ParseAndSplit(request.getSqlProgram());
       } else {
-        HiveQLParserAdaptor parser = new HiveQLParserAdaptor();
+        ParseInterface parser = new HiveQLParserAdaptor();
         parse_result = parser.ParseAndSplit(request.getSqlProgram());
```

Comment:
parseResult

Reply:
The Java style check in our pre-commit hook doesn't complain about the variable naming though. Instead of spotting the naming manually, a better approach is configuring the style checker.

Also, if we decide to go with the camel case, there are multiple variable names needs to be changed (e.g. `sub_sql`, `accumulated_position`) in addition to `parser_result`, so it is better to file the change in an separate PR.